### ISSUE TITLE
panic:Cannot convert object to primitive value

### ISSUE THREAD

**Issue by relsunkaev** [pkgxdev/pkgx#476]
Encountered a panic when trying to install python 3.11.2
```
python.org=3.11.2 cache: hit
python.org/v3.11.2/lib/python3.11/config-3.11-darwin/libpython3.11.a: Lzma library error:  No progress is possible
tar: Error exit delayed from previous errors.
```
Here's the attachment
```
TypeError: Cannot convert object to primitive value
    at install (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/prefab/install.ts:91:55)
    at async install (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/hooks/useExec.ts:117:25)
    at async default (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/hooks/useExec.ts:79:38)
    at async file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/app.ts:35:63
```

**Comment by getneil** [comment]
probabaly same
```
TypeError: Cannot convert object to primitive value
    at install (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/prefab/install.ts:91:55)
    at async install (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/hooks/useExec.ts:117:25)
    at async default (file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/hooks/useExec.ts:79:38)
    at async file:///private/var/folders/y3/yr3bxsy95c3gfwwj61ytbp000000gq/T/998ad54c/src/app.ts:35:63
```

**Comment by mxcl** [comment]
I believe this happens when an install is canceled and the tarball is not a complete download and then you later try again.

This is fixed in #492 and deployed as 0.26.1, please `rm -rf ~/.tea/tea.xyz/var/www` as well.

**Comment by mxcl** [comment]
There error message is also fixed, should say `tar failed with exit status 1`
